app/src/main/java/edu/agh/susgame/front/ui/MainActivity.kt

```diff
@@ -31,11 +36,13 @@ class MainActivity : ComponentActivity() {
                 ) {
                     val navController = rememberNavController()
 
-                    MenuNavigationHost(
-                        navController,
-                        serverMapProvider,
-                        awaitingGamesProvider,
-                    )
+                    Box(modifier = Modifier.padding(PaddingL)) {
```

Comment:
![image](https://github.com/Nepommuck/SusGameFrontend/assets/100963527/ca919153-5a4b-449d-8a70-9c9f1bb4982f)

w sumie nie ma to znaczenia bo zahacza to o wystylizowanie apki, ale piszę zeby potem nie szukac przyczyn, przez ten padding jest malo miejsca dla guzika, po zmianie na S juz wraca do normy

Reply:
A wybacz, że zignorowałem. Klikałem z telefonu i myślałem, że nie ma więcej komentarzy, a było "Approved"

Hmm faktycznie to się psuje, gdy jest mały ekran i pionowo, ale z tego co widzę, inne widoki też średnio dają radę:

![image](https://github.com/Nepommuck/SusGameFrontend/assets/100998310/2cfd100d-84aa-49e5-8d52-66c200ea4f3f)

Proponuję na razie to zostawić i najwyżej w przyszłości postarać się, żeby cała apka ładnie się wyświetlała na małych telefonach

Reply:
W sumie ten wymóg, że ma to spoko działać zarówno pionowo jak i poziomo dużo utrudnia. Zostawiłbym to na razie i przedyskutował w przyszłości, czy nie zrobić może całej gry tylko poziomej